newIDE/app/src/ResourcesList/index.js

```diff
@@ -298,7 +296,7 @@ export default class ResourcesList extends React.Component<Props, State> {
     ];
   };
 
-  componentDidMount() {
+  forceCheckMissingPaths = () => {
```

Comment:
nitpicking: there is nothing "forcing" here right? You can call it "checkMissingPaths" instead I think

Reply:
Well there is the forceUpdateList, but it's the update that is forced (because otherwise the list is unaware that changes have been made), it's not the check that is forced :)